src/com/shashi/service/impl/SellerServiceImpl.java

```diff
@@ -0,0 +1,167 @@
+package com.shashi.service.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.shashi.beans.ProductBean;
+import com.shashi.service.SellerService;
+import com.shashi.utility.DBUtil;
+
+public class SellerServiceImpl implements SellerService {
+
+	@Override
+	public List<ProductBean> getAllProductsBySeller(String companyName) {
+		List<ProductBean> products = new ArrayList<ProductBean>();
+
+		Connection con = DBUtil.provideConnection();
+
+		PreparedStatement ps = null;
+		ResultSet rs = null;
+
+		try {
+			ps = con.prepareStatement("select * from product where sellerId=?");
+			ps.setString(1, companyName);
```

Comment:
what does this line do? I thought the ``1`` was for the column but it seems like it's for something else.

Reply:
oh never mind, it's to replace the ``?`` in the sql query.